pages/events/summer-school.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <main class="summer-school-page">
+    <AppPageHeaderWithCard
+      :description="eventData.header.description"
+      :cta="eventData.header.cta"
+      :card-title="eventData.header.cardSectionHeading"
+    >
+      <template slot="title">
+        {{ eventData.header.primaryTitle }}
+        <br>
+        {{ eventData.header.secondaryTitle }}
+      </template>
+      <template slot="card">
+        <EventCard v-bind="eventData.header.card" vertical-layout>
+          {{ eventData.header.card.description }}
+        </EventCard>
+      </template>
+    </AppPageHeaderWithCard>
+
+    <AppMosaicSection
+      class="summer-school-page__section"
+      :title="eventData.mosaic.title"
+      :elements="eventData.mosaic.elements"
+    />
+
+    <section class="summer-school-page__section">
+      <h2 class="copy__title">
+        {{ eventData.agenda.title }}
+      </h2>
```

Comment:
Just as a _nice to have_, we could go for a one-liner here:

```suggestion
      <h2 class="copy__title" v-text="eventData.agenda.title" />
```